dotcom-rendering/src/lib/messagePicker.ts

```diff
@@ -42,8 +42,11 @@ const recordMessageTimeoutInOphan = async (
 ) => {
 	const ophan = await getOphan(renderingTarget);
 	ophan.record({
-		component: `${slotName}-picker-timeout-dcr`,
-		value: candidateId,
+		componentEvent: {
+			// @ts-expect-error -- Type 'string' is not assignable to type 'ComponentV2'.
```

Comment:
I don't _think_ anyone is looking at this data. I'll confirm with my team but I think this could be removed (along with the other lines where tx-engineers have been tagged).